docs-conceptual/azps-4.8.0/new-azureps-module-az.md

```diff
@@ -101,7 +101,7 @@ compatibility aliases across PowerShell sessions. For more information, see
 > migration for you! See the [full breaking changes list](migrate-az-1.0.0.md) to find where your
 > scripts may require updates.
 
-## Continued support for AzureRM
+## Support for AzureRM
 
-AzureRM will no longer receive new cmdlets or features. However, the AzureRM module is still
-officially maintained and will get bug fixes through December 2020.
\ No newline at end of file
+All versions of the AzureRM PowerShell module will be retired on February 29, 2024 and will no
+longer receive critical fixes.
```

Comment:
Because Az PowerShell modules now have all the capabilities of AzureRM PowerShell modules and more, we'll retire AzureRM PowerShell modules on 29 February 2024.

To avoid service interruptions, [update your scripts](http://aka.ms/azpsmigrate)  that use AzureRM PowerShell modules to use Az PowerShell modules by 29 February 2024. To automatically update your scripts, follow the [quickstart guide](https://docs.microsoft.com/powershell/azure/quickstart-migrate-azurerm-to-az-automatically).